lib/resolveLang.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var defaultLangCode = require('./defaultLangCode'),
+	normalizeLangCode = require('./normalizeLangCode');
+
+var locales = [
```

Comment:
You're right the first part should be "languages", I think the second part is "countries" though. [RFC 3066](https://www.ietf.org/rfc/rfc3066.txt) makes several references to "country code" as the subtag.